### ISSUE TITLE
Ruuvi Measurements Missing

### ISSUE THREAD

**Issue by arctechllc** [mwaylabs/fruitymesh#173]
I'm building out a wireless sensor network to track atmospheric readings.  One use case is custody chain for covid vaccines, and there are many others. 

I'm using RuuviTags as the sensor nodes, I'm using an esp32 M5Stack for a receiver, and I have a full backend I built on AWS using Timestream to house all readings. 

As it stands, the system works great so long as the tags are within range of an esp32 gateway device.  This is not always practical, however, and a mesh system would be very useful here so you can use only 1 gateway and let the tags mesh with one another (they are close to one another, but not necessarily to the gateway).

I flashed 2 new RuuviTags with the latest release Ruuvi merge firmware from your repository.  When I listen to either tag, I see 2 different advertisements over and over again, but neither one appears to change whatsoever so it's highly unlikely there is sensor data in them.

Moreover, when I use the RuuviStation app, it only updates once every 3 min if that, and the mesh itself doesn't appear to be working because I can only see sensors in the app which are within range. 

I'm using the latest version of RuuviTags (delivered yesterday).  Please see below the output from BLE listener:

---

17:53:19.360 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-97,"p":"0201061BFF4D02F00B0001690669060EE00200121C0401FFFF100000000000"}}
17:53:20.015 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-102,"p":"0201061BFF4D02F00B0001690669060EE00200121C0401FFFF100000000000"}}
17:53:26.009 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-99,"p":"0201061BFF4D02F00B0001690669060EE00200121C0401FFFF100000000000"}}
17:53:28.160 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-95,"p":"020106030312FE171612FE03000B0009961C29000000000100000000000000","0xfe12":"03000B0009961C29000000000100000000000000"}}
17:53:34.165 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-103,"p":"0201061BFF4D02F00B0001690669060EE00200121C0401FFFF100000000000"}}
17:53:34.461 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-99,"p":"020106030312FE171612FE03000B0009961C29000000000100000000000000","0xfe12":"03000B0009961C29000000000100000000000000"}}
17:53:36.461 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-99,"p":"020106030312FE171612FE03000B0009961C29000000000100000000000000","0xfe12":"03000B0009961C29000000000100000000000000"}}
17:53:37.819 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-98,"p":"0201061BFF4D02F00B0001690669060EE00200121C0401FFFF100000000000"}}
17:53:44.572 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-101,"p":"020106030312FE171612FE03000B0009961C29000000000100000000000000","0xfe12":"03000B0009961C29000000000100000000000000"}}
17:53:46.668 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-98,"p":"0201061BFF4D02F00B0001690669060EE00200121C0401FFFF100000000000"}}
17:53:47.619 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-101,"p":"020106030312FE171612FE03000B0009961C29000000000100000000000000","0xfe12":"03000B0009961C29000000000100000000000000"}}
17:53:47.972 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-99,"p":"020106030312FE171612FE03000B0009961C29000000000100000000000000","0xfe12":"03000B0009961C29000000000100000000000000"}}
17:53:48.273 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-98,"p":"020106030312FE171612FE03000B0009961C29000000000100000000000000","0xfe12":"03000B0009961C29000000000100000000000000"}}
17:53:48.992 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-97,"p":"0201061BFF4D02F00B0001690669060EE00200121C0401FFFF100000000000"}}
17:53:51.015 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-101,"p":"0201061BFF4D02F00B0001690669060EE00200121C0401FFFF100000000000"}}
17:53:51.310 MQT: tele/tasmota_454778/BLE = {"details":{"mac":"C31710AC92AF","RSSI":-98,"p":"0201061BFF4D02F00B0001690669060EE00200121C0401FFFF100000000000"}}

**Comment by siretty** [comment]
Hi @arctechllc, in order to analyze your issue we will need some details on your setup.

- Your RuuviTags would be hardware revision B7.1, correct? You can find this revision number on the back of the circuit board in gold-plated copper next to the vias under the nRF chip. B7.1 is also the version I tested the firmware on.
- How many tags did you use for your test?
- How far away are the tags from each other and how far away were you from the nearest tag?
- How long did you wait before you recorded the advertisments?

The advertisments you did receive are not sensor data, but used by FruityMesh internally. The manufacturer id for Ruuvi Innovations Ltd. is [0x99, 0x04] (lsb), but the advertisments you see have manufacturer id [0x4D, 0x02] (lsb) which is M-Way Solutions GmbH.

Just going by your initial post and a hunch, it is likely that you are not close enough to the tags to receive the sensor data and potentially that the tags are not close enough to each other to establish a mesh.

Could you try to place at least two or three tags right in front of you and see if the mesh forms and the app picks up the sensor data from all of the tags?

**Comment by arctechllc** [comment]
Hi @siretty

Thank you for the prompt response.

Indeed the version is b7.1

Im using 2 tags to test the mesh, and they are right next to each other in front of me about 30cm from my phone and 1m line of sight from the esp32

I tried to manually reset them a couple of times with a view that they may reestablish a link.

I will now reflash both and try again just in case something happened during flash process. I'm using Ruuvi shield to do this

Will update if anything changes my end

**Comment by siretty** [comment]
Hello @arctechllc, just a quick heads up before you run into another issue yourself. There is a known issue with the RuuviWeatherModule in FruityMesh that leads to fairly high current consumption (drains batteries rather quickly at the moment). I'm actively working on that, just so you know.

**Comment by arctechllc** [comment]
Hi @siretty


Many thanks for the information. I have not had time to dig into my setup but I did all the basic troubleshooting to no avail.

I'll wait to see the new commit. Good luck!

**Comment by siretty** [comment]
Hey @arctechllc, attached is a debug build of the firmware.

It has the LED mode set to CONNECTIONS (so tags flashed with this firmware will blink red when not connected at all and if they are connected the number of green flashes indicates the number of connections). Also the terminal is accessible via JLinkRTTViewer, so you can see if any errors are generated. I also noticed that silly old me had set the transmission power of the sensor advertisments to -40dBm which may be related to you not receiving them (this was _really_ short range). This debug firmware has it set to +4dBm (which for me goes through at least two walls) (RuuviWeatherModule.cpp:110).

FYI: The sensor measurements are (on a single tag) generated in 30s intervals (this can be changed in the firmware though).

[2021-03-19-18-47_prod_ruuvi_weather_nrf52_merged_debug_with_leds_and_terminal.hex.zip](https://github.com/mwaylabs/fruitymesh/files/6173234/2021-03-19-18-47_prod_ruuvi_weather_nrf52_merged_debug_with_leds_and_terminal.hex.zip)

**Comment by arctechllc** [comment]
@siretty Mate, you're awesome. I really appreciate this. Let me give it some tests and I'll come back on here

**Comment by arctechllc** [comment]
Hi @siretty 

Thanks again for providing the debug firmware.  I can confirm the ruuvitags connect to one another as they both blink green once.  My sniffer picks up all the advertisements, but I cannot seem to pick up data from one by listening to the other.  Maybe I've misunderstood how this works?  Would the advertisement including data from other mesh nodes look different?

Here is a sample output from one of the 2 ruuvitags running the above firmware.  I bolded one of the lines that shows the readings from the tag whose mac I am sniffing, but I see nothing parallel for the other mac (E2:17:78:01:EF:F9)

Many thanks for your time in advance.

03:38:26.692 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-62,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:27.042 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:27.242 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:27.392 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-71,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:27.542 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:27.843 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:28.193 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:28.493 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-71,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:29.342 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-62,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:29.492 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:29.641 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:29.792 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:29.992 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-62,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:30.293 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-62,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:30.593 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:31.244 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:31.442 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:31.592 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-71,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:31.792 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:31.892 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-71,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:32.206 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:33.171 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:33.371 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-68,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:33.471 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:33.671 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-68,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:34.021 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-68,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:34.320 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:34.520 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:34.621 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:34.820 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:34.921 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:39.015 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:39.215 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:39.315 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:39.565 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:39.665 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:39.865 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:39.965 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-71,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:40.816 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:41.017 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:41.117 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:41.317 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:41.467 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:41.617 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:41.967 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:42.316 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:42.617 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:42.917 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:43.218 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:43.867 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:44.217 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:44.817 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:45.450 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:45.650 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:46.002 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:46.202 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:46.302 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:46.502 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:46.802 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:47.004 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:47.352 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:47.451 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:47.804 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:48.267 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:48.417 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:48.917 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:49.067 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:49.267 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:49.567 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:49.867 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:50.067 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:50.217 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:53.114 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:53.264 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:53.465 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:53.765 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:53.965 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:54.068 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:54.265 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:54.414 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF99040511CA527BC8BF800080008000C816FF000FF438B7A648DC"}}
03:38:54.515 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-68,"p":"0201061BFF99040511CA527BC8BF800080008000C816FF000FF438B7A648DC"}}
03:38:54.565 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:54.915 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:55.215 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:55.414 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:55.515 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:55.865 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:56.064 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:56.364 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:56.564 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"lost":true,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:56.714 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:56.864 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:57.165 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:57.415 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:58.383 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:58.883 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:59.183 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:38:59.383 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:59.483 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:38:59.833 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:00.133 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:00.333 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:00.598 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:01.111 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:01.311 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-71,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:01.411 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:01.611 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:01.710 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:01.810 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:02.011 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:02.160 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:02.311 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:02.460 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:02.662 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:02.961 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:03.264 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:03.322 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:03.472 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:03.572 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:03.822 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:04.122 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:04.772 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:05.072 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:05.272 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:05.372 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:05.572 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:05.672 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:10.773 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:10.923 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:11.073 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:11.273 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:11.372 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:11.574 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:11.724 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:11.924 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:12.874 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:13.074 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:13.173 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:13.374 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:13.674 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-69,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:13.874 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:14.174 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:14.324 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:14.524 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:14.824 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:14.974 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:15.174 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:15.474 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:15.941 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-68,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:15.000 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"lost":true,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:16.099 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:16.300 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:16.749 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:16.950 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:17.250 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:17.450 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:17.550 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:17.750 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:17.900 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:18.100 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:18.199 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:19.013 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:19.363 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:19.562 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-65,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:19.662 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:19.863 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:20.013 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:20.163 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:20.513 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:20.813 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:21.858 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:22.207 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:22.309 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:22.507 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:22.808 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:23.158 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-65,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:23.358 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:23.458 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:23.658 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-71,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:23.947 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-68,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:24.007 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:24.307 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:24.757 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-69,"p":"0201061BFF99040511C252E1C8C2800080008000C816FF0010F438B7A648DC"}}
03:39:25.109 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:25.458 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:25.658 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:25.758 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
**03:39:25.958 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-69,"p":"0201061BFF99040511C252E1C8C2800080008000C816FF0010F438B7A648DC"}}**
03:39:26.258 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-68,"lost":true,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:39:26.929 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:26.980 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"0201061BFF99040511C252E1C8C2800080008000C816FF0010F438B7A648DC"}}
03:39:27.080 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:27.230 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:27.430 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-68,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:27.530 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-68,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:27.730 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-72,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:28.080 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:28.180 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-71,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:39:28.680 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:02.314 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:02.614 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:02.865 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:02.965 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-63,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:03.164 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:03.465 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:03.765 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:03.965 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-73,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:05.115 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:05.266 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:05.466 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:05.766 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:06.116 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-69,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:06.266 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:06.616 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:06.916 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:07.266 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-69,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:07.566 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:07.866 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:08.066 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:08.416 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-65,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:08.516 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:10.183 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:10.483 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-66,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:10.683 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:10.833 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:10.983 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-69,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:11.134 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:11.635 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:11.785 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-69,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:11.936 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:12.084 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:13.983 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:14.182 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-65,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:14.483 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:14.833 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:15.033 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:15.132 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:15.333 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:15.432 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:15.633 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:16.783 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:17.133 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:17.433 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:17.733 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:17.933 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-64,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:18.083 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-65,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:18.284 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:18.382 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:18.583 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-67,"p":"020106030312FE171612FE03000B0009E12929000000000100000000000000","0xfe12":"03000B0009E12929000000000100000000000000"}}
03:40:18.883 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:19.233 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-70,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}
03:40:19.432 RSL: tele/tasmota_8C806C/BLE = {"details":{"mac":"F438B7A648DC","RSSI":-65,"p":"0201061BFF4D02F00B00011604BA00AC56020019200401FFFF100000000000"}}

**Comment by siretty** [comment]
Hello @arctechllc,

I am very sorry for taking this long to find time to reply to your last comment. I hope you've been doing well.

In the meantime I have tried to reproduce your issue multiple times on different devices, but for me the connection is established and sensor data advertisments are generated as expected.

Would you be so kind as to show the command you used for flashing, and if you are able to access the log of at least one of the nodes available via JLinkRTTViewer when using the firmware from [my previous comment](https://github.com/mwaylabs/fruitymesh/issues/173#issuecomment-803012450)? There is a log message which is printed when sensor data is received from another node - which we can use to see if the problem is in the advertisment functionality, between the nodes or in the code reading and distributing the sensor data.

**Comment by arctechllc** [comment]
Hi @siretty ,


Sorry it took me a while to come back. I just noticed the new release so I will set up a new demo network and try again. I think I may have been looking for the mesh messaging in the advertised payloads but this actually requires a direct connection to one of the nodes. Kind of an amateurish error on my end, but I'm still fairly new to this technology in general so I'm learning as I go. Do you have a preferred bluetooth sniffer app/hardware that you use to test whether the mesh is working or not? I will use JLink eventually but I'm waiting on some hardware to do that

**Comment by mariusheil** [comment]
Hello, I'll close this as there has not been any more feedback. Feel free to reopen it.
